moped-editor/src/views/projects/projectView/ProjectComponents/EditAttributesModal.js

```diff
@@ -50,8 +50,11 @@ const EditAttributesModal = ({
   const onSave = (formData) => {
     const isSavingSignalFeature = Boolean(formData.signal);
 
-    const { description, subcomponents, phase, subphase, tags } = formData;
+    const { subcomponents, phase, subphase, tags } = formData;
     const completionDate = !!phase ? formData.completionDate : null;
+    const description =
+      formData.description.length > 0 ? formData.description : null;
```

Comment:
This is a bug fix. Description could save as an empty string when edited in the existing code.